aws-api/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            returnDefaultValues = true
```

Comment:
**Let's not add this.**

> Caution: Setting the returnDefaultValues property to true should be done with care. The null/zero return values can introduce regressions in your tests, which are hard to debug and might allow failing tests to pass. Only use it as a last resort.

maybe you added this before you put `@RunWith(RobolectricTestRunner.class)` on your test class? That should be enough, no?

Reply:
you are right!